### PR TITLE
Python Examples

### PR DESCRIPTION
**Pull request by pavankumarbn**

### CODE REVIEW

File: src/Python/SimpleOperations/UniformRandomNumber.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+import vtk
+import vtkmath
+def main():
+    numRand = 3;
+    #Generate numRand random numbers from a uniform distribution between 0.0 and 2.0
+    for i in range(0,numRand):
+    	i = vtkmath.random.uniform(0.0,2.0)
+    	print(i)
```
Comment (ajpmaclean) [inline src/Python/SimpleOperations/UniformRandomNumber.py]: random is no longer recommended use [vtkMinimalStandardRandomSequence](https://vtk.org/doc/nightly/html/classvtkMinimalStandardRandomSequence.html) and [vtkBoxMuellerRandomSequence](https://vtk.org/doc/nightly/html/classvtkBoxMuellerRandomSequence.html) I will be removing random from existing code. Do look at existing python examples to see how it is used.

Comment (pavankumarbn) [reply src/Python/SimpleOperations/UniformRandomNumber.py]: @ajpmaclean Sure!!

**Comment by ajpmaclean** [comment]
At this point I am working through doing a major cleanup all the existing Python code so please do not update existing code until I have finished, which should be in a few days. So you should be seeing many changes as things progress.

**Comment by pavankumarbn** [comment]
@ajpmaclean Sure, I will not update any examples. Thank you.

**Comment by ajpmaclean** [comment]
Shouldn't be too long, I've just completed down to the PolyData folder.

**Comment by pavankumarbn** [comment]
@ajpmaclean Ok

**Pull request closed without merging**